package/MDAnalysis/topology/RDKitParser.py

```diff
@@ -177,8 +177,16 @@ def parse(self, **kwargs):
         occupancies = []
         tempfactors = []
 
+        try:
+            atom = mol.GetAtomWithIdx(0)
+        except:
```

Comment:
As the auto-linter suggests, you may want to be more specific here so that you don't accidentally intercept other important exceptions.

Looking at the matching issue, you may be able to use `except RuntimeError` to be more specific.

Reply:
Hi Tyler, I fixed that as soon as I have done the pull request